flycheck.el

```diff
@@ -9890,10 +9903,20 @@ See URL `http://call-cc.org/'."
                                  (zero-or-more not-newline))
                    (one-or-more space) "<--")
           line-end)
+   ;; A of version 4.12.0, the chicken compiler doesn't provide a
+   ;; line number for this error.
+   (error line-start "Syntax error: "
```

Comment:
So, to be clear: the pattern above this one shows there are some syntax errors which *do* have line numbers, while some other do not?

Reply:
That's correct...

Reply:
I just wanted to make sure, as it seems a strange behavior.  But at least now we can report these error to the user.